examples/coin_flipping/coin_flipping.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+
+from objects.qubit import Qubit
+from components.host import Host
+from components.network import Network
+from backends.eqsn_backend import EQSNBackend
+# from backends.cqc_backend import CQCBackend
+# from backends.projectq_backend import ProjectQBackend
+
+
+def quantum_coin_flipping(host, m, n, partner_id, rot_angle):
+    """
+    Quantum Coin Flipping Protocol.
+
+    Own random variables are b_j and d_ij.
+    Partners random variables are a_j and c_ij.
+
+    Own shared: f_ij = b_j ^ d_ij
+    Partner shared: e_ij = a_j ^ c_ij
+
+    final own: B = ^b_j for all j
+    final partner: A_tilde = ^a_j for all j
+
+    final random bit: A_tilde ^ B
+    """
+    b = np.random.randint(0, 2, m, dtype=int)
```

Comment:
For these variable names `b, d, a`, is there another variable name that is maybe redundant but obvious in what it represents? I'd say the goal of these examples is to be able to read the code and understand what the protocol is doing without knowing it in advance.

Reply:
Maybe a comment also works here, but one advice I got is to try to have cleaner code instead of adding comments.

Reply:
I adapted the names and added some comments